CODEOWNERS

```diff
@@ -1,13 +1,16 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
+# Argo Workflows
+/charts/argo @benjaminws @stefansedich @paguos
+
 # Argo CD
 /charts/argo-cd @seanson @spencergilbert
 
 # Argo Events
 /charts/argo-events @jbehling
 
-# Argo Workflows
-/charts/argo @benjaminws @stefansedich @paguos
-
 # Argo Rollouts
 /charts/argo-rollouts @cabrinha
+
+# Argo CD Notifications
+/charts/argocd-notifications @alexmt @andyfeller
```

Comment:
Should this list match the list defined in `Chart.yaml`?
https://github.com/argoproj/argo-helm/pull/365/files#diff-c10017cc8b44885eb18634170c26c1aeR14